### DISCUSSION TITLE
How to install WSA 2308?

### DISCUSSION THREAD

**Discussion by manu2551988** [LSPosed/MagiskOnWSALocal#693] [Q&A] [answered]
Can you please let me know how to install WSA 2308 or what need to be modified in script to download WSA 2308

**Comment by Almighty188** [comment, marked as answer]
See https://github.com/LSPosed/MagiskOnWSALocal/discussions/691

**Reply by manu2551988** [reply]
Thanks for pointing to right direction.

**Reply by Almighty188** [reply]
You are very welcome....  Let me try explaining it here in more detail...

Generally, you would want to follow https://github.com/LSPosed/MagiskOnWSALocal with the section Text Guide and note the following...
Step 2 as listed should only be done only once on a system to clone the repository
so basically only do steps 2 and then 3 the first time
Afterwards, start from step 3 first followed by:
sudo git pull 
and before you get to 4, use the instructions from my link above for 2308... 2309 is already out so give me maybe 15-20 minutes and you will see a link for that.  The instructions don't really change except the download link is obviously different for different versions... then you just proceed with step 4

**Reply by manu2551988** [reply]
I already rooted WSA 2309 using this work excellently. Thanks, a lot. It helps me a lot with testing apps using WSA with Google Play Services. I have 1st hand experience with testing this product often.

**Reply by Almighty188** [reply]
Yes, just wanted to make sure you had the correct instructions as it's hard to tell what level of experience the other person has so better to make sure they understood everything completely.
